addons/viewport/src/Tool.tsx

```diff
@@ -195,12 +196,15 @@ export const ViewportTool: FunctionComponent<{}> = React.memo(
                 },
                 [`#${wrapperId}`]: {
                   padding: theme.layoutMargin,
-                  display: 'flex',
                   alignContent: 'center',
                   alignItems: 'center',
                   justifyContent: 'center',
```

Comment:
if its not a flex container do we still need justify content and align items?

Reply:
`display: grid`